pkg/api/const.go

```diff
@@ -434,10 +435,23 @@ const (
 const (
 	//AzureADIdentitySystem Identity System
```

Comment:
Picky. MS documentation refers to `AAD` and `ADFS` as `Identity Providers`, not `Identity Systems`

https://docs.microsoft.com/en-us/azure/azure-stack/azure-stack-identity-overview

Reply:
I will ask PM for this and address it in the following changes.
In the getazuretsackstampinfo cmdlet in Azure Stack, they are AzureAD and ADFS.
the convension in aks engine for other parameters is using _ as seperator and lower case

Reply:
I did not mean to say that we should use " " as separator or change the case. I am just pointing out that it may be more accurate to use the term `provider` over `system`. Just a small thing.

Reply:
I see, in getazuretsackstampinfo cmdlet, it is IdentitySystem. let me check with PM.